doc/manual.md

```diff
@@ -150,7 +150,7 @@ all dots to path separators, and then appending `.so`, for binary modules, or
 modules will correspond to `foo.{so|titan}`, `foo/bar.{so|titan}`, and `foo/bar/baz.{so|titan}`.
 The Titan compiler will recompile the module if its source is newer than its binary.
 
-Binary modules are looked up in the *runtime search path*, a semicolon-separated list
```

Comment:
Is this one supposed to be runtime or run-time?

Reply:
run-time

Sorry, it looks like I forgot this one.